src/test/java/nextstep/member/unit/GithubClientTest.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.member.unit;
+
+import nextstep.member.domain.GithubClient;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.assertj.core.api.Assertions.*;
+
+// 테스트 컨트롤러의 포트가 고정되어 있으므로 DEFINED_PORT 사용
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DisplayName("깃허브 클라이언트 테스트")
+public class GithubClientTest {
```

Comment:
우석님께서 의도한 바가 아니라면 아래와 같이 다른 인수 테스트 파일이 상속받는 `AcceptanceTest`를 `GithubClientTest`도 상속 받으면 어떨까요? 
말씀하신 것 처럼 `GithubClientTest`은 다른 인수 테스트 파일과 형상이 달라 @ActiveProfiles("test") 붙지 않은 경우, 혼자 다른 프로파일 형상을 갖게 되어 컨테이너가 2개 띄어져 발생한 문제라고 판단됩니다.
```suggestion
@DisplayName("깃허브 클라이언트 테스트")
public class GithubClientTest  extends AcceptanceTest  {
```

Reply:
맨 처음에는AcceptanceTest를 상속 받아서 해결했었는데,깃허브 클라이언트 클래스는 인수 테스트가 아니지 않나? 라는 생각 때문에 별도로 프로필만을 적용했었어요.
승연님은 혹시 이 로직이 인수 테스트에 해당한다고 생각하시는 것일까요?

Reply:
앗, 해당 테스트는 인수 테스트가 아니죠 😂 제가 꼼꼼히 보질 못해 죄송해요 🥲
그러면 전체 테스트가 하나의 프로파일을 통해 수행되도록 하는 것만 체크해보면 좋을 것 같아요! 
추가로 테스트 설정 파일을 두어 개발/테스트 간의 환경 설정 분리를 해주는 것을 추천드립니다:)

Reply:
음..말씀해 주신 부분이 잘 이해가 되지 않는 것 같아요..

'전체 테스트가 하나의 프로파일을 통해 수행되도록 하는 것만 체크'는 지금처럼 모든 테스트가 동일한 프로필로 구동되는지 일일히 체크해야 한다는 말씀이 맞을까요?

그리고 '테스트 설정 파일을 두어 개발/테스트 간의 환경 설정 분리'는 https://blogshine.tistory.com/527 이런 방법을 말씀하시는 거겠죠..?